packages/harmony/src/components/typography/Icons/Icon.module.css

```diff
@@ -0,0 +1,33 @@
+.icon {
+  --icon-color: currentColor;
+  --icon-size: var(--harmony-unit-4);
+  width: var(--icon-size);
+  min-width: var(--icon-size);
+  height: var(--icon-size);
+}
+
+.icon path {
+  fill: var(--icon-color);
+}
+
+.xSmall {
+  --icon-size: calc(3.5 * var(--unit));
+}
+.small {
+  --icon-size: var(--harmony-unit-4);
+}
+.medium {
+  --icon-size: var(--harmony-unit-5);
+}
+.large {
+  --icon-size: var(--harmony-unit-6);
+}
+.xLarge {
+  --icon-size: calc(7.5 * var(--harmony-unit));
+}
+.xxLarge {
```

Comment:
our other harmony size names use `2xL`

Reply:
Nice call, ported this from the icon Randy wrote, so let's update it with all the new standard keys

Reply:
headsup, going to update this when we get the emotion theme standardized so we can just reference types from there